services/dex/2.15.0/defaults/cm.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: dex-2.14.0-d2iq-defaults
+  name: dex-2.15.0-d2iq-defaults
```

```suggestion
  name: dex-2.14.1-d2iq-defaults
```

Lets do 2.14.1 as this is packaged under `dex` and that one doesn't have 2.15.x version yet.